 Chromatin Immunoprecipitation (ChIP) is a type of immunoprecipitation experimental technique used to investigate the interaction between proteins and DNA in the cell. It aims to determine whether specific proteins are associated with specific genomic regions, such as transcription factors on promoters or other DNA binding sites, and possibly defining cistromes. ChIP also aims to determine the specific location in the genome that various histone modifications are associated with, indicating the target of the histone modifiers. Briefly, the method is as follows: protein and associated chromatin in a cell lysate are temporarily bonded, the DNA-protein complexes (chromatin-protein) are then sheared and DNA fragments associated with the protein(s) of interest are selectively immunoprecipitated, and the associated DNA fragments are purified and their sequence is determined. These DNA sequences are supposed to be associated with the protein of interest in vivo. There are mainly two types of ChIP, primarily differing in the starting chromatin preparation. The first uses reversibly cross-linked chromatin sheared by sonication called cross-linked ChIP (XChIP). Native ChIP (NChIP) uses native chromatin sheared by micrococcal nuclease digestion. Cross-linked ChIP is mainly suited for mapping the DNA target of transcription factors or other chromatin-associated proteins, and uses reversibly cross-linked chromatin as starting material. The agent for reversible cross-linking could be formaldehyde or UV light. Then the cross-linked chromatin is usually sheared by sonication, providing fragments of 300 - 1000 base pairs (bp) in length. Mild formaldehyde crosslinking followed by nuclease digestion has been used to shear the chromatin. Chromatin fragments of 400 - 500bp have proven to be suitable for ChIP assays as they cover two to three nucleosomes. Cell debris in the sheared lysate is then cleared by sedimentation and protein–DNA complexes are selectively immunoprecipitated using specific antibodies to the protein(s) of interest. The antibodies are commonly coupled to agarose, sepharose or magnetic beads. The immunoprecipitated complexes (i.e., the bead–antibody–protein–target DNA sequence complex) are then collected and washed to remove non-specifically bound chromatin, the protein–DNA cross-link is reversed and proteins are removed by digestion with proteinase K. An epitope-tagged version of the protein of interest, or in vivo biotinylation can be used instead of antibodies to the native protein of interest. The DNA associated with the complex is then purified and identified by polymerase chain reaction (PCR), microarrays (ChIP-on-chip), molecular cloning and sequencing, or direct high-throughput sequencing (ChIP-Seq). Native ChIP is mainly suited for mapping the DNA target of histone modifiers. Generally, native chromatin is used as starting chromatin. As histones wrap around DNA to form nucleosomes, they are naturally linked. Then the chromatin is sheared by micrococcal nuclease digestion, which cuts DNA at the length of the linker, leaving nucleosomes intact and providing DNA fragments of one nucleosome (200bp) to five nucleosomes (1000bp) in length. Thereafter, methods similar to XChIP are used for clearing the cell debris, immunoprecipitating the protein of interest, removing protein from the immunoprecipated complex, and purifying and analyzing the complex-associated DNA. The major advantage for NChIP is antibody specificity. It is important to note that most antibodies to modified histones are raised against unfixed, synthetic peptide antigens and that the epitopes they need to recognize in the XChIP may be disrupted or destroyed by formaldehyde cross-linking, particularly as the cross-links are likely to involve lysine e-amino groups in the N-terminals, disrupting the epitopes. This is likely to explain the consistently low efficiency of XChIP protocols compare to NChIP. But XChIP and NChIP have different aims and advantages relative to each other. XChIP is for mapping target sites of transcription factors and other chromatin associated proteins; NChIP is for mapping target sites of histone modifiers (see Table 1). Table 1 Advantages and disadvantages of NChIP and XChIP In 1984 John T. Lis and David Gilmour, at the time a graduate student in the Lis lab, used UV irradiation to covalently cross-link proteins in contact with neighboring DNA in intact living bacterial cells. Following lysis of cross-linked cells and immunoprecipitation of bacterial RNA polymerase, DNA associated with enriched RNA polymerase was hybridized to probes corresponding to different regions of known genes to determine the in vivo distribution and density of RNA polymerase at these genes. A year later they used the same methodology to study distribution of eukaryotic RNA polymerase II on fruit fly heat shock genes. These reports could perhaps be considered the pioneering work in the field of chromatin immunoprecipitation. XChIP was further modified and developed by Alexander Varshavsky and co-workers, who examined distribution of histone H4 on heat shock genes using formaldehyde cross-linking. This technique was extensively developed and refined thereafter. NChIP approach was first described by Hebbes et al., 1988, and also been developed and refined quickly. The typical ChIP assay usually take 4–5 days, and require 106~ 107 cells at least. Now new techniques on ChIP could be achieved as few as 100~1000 cells and complete within one day. ChIP has also been applied for genome wide analysis by combining with microarray technology (ChIP-on-chip) or second generation DNA-sequencing technology (Chip-Sequencing). ChIP can also combine with paired-end tags sequencing in Chromatin Interaction Analysis using Paired End Tag sequencing (ChIA-PET), a technique developed for large-scale, de novo analysis of higher-order chromatin structures. 
