 The Human Microbiome Project (HMP) is a United States National Institutes of Health initiative with the goal of identifying and characterizing the microorganisms which are found in association with both healthy and diseased humans (the Human microbiome). Launched in 2008, it is a five-year project, best characterized as a feasibility study, and has a total budget of $115 million. The ultimate goal of this and similar NIH-sponsored microbiome projects is to test how changes in the human microbiome are associated with human health or disease. This topic is currently not well understood. Important components of the Human Microbiome Project will be culture-independent methods of microbial community characterization, such as metagenomics (which provides a broad genetic perspective on a single microbial community), as well as extensive whole genome sequencing (which provides a "deep" genetic perspective on certain aspects of a given microbial community, i.e. of individual bacterial species). The latter will serve as reference genomic sequences — 3000 such sequences of individual bacterial isolates are currently planned — for comparison purposes during subsequent metagenomic analysis. The microbiology of five body sites will be emphasized: oral, skin, vaginal, gut, and nasal/lung. The project also is financing deep sequencing of bacterial 16S rRNA sequences amplified by polymerase chain reaction from human subjects. Total microbial cells found in association with humans may exceed the total number of cells making up the human body by a factor of ten-to-one. The total number of genes associated with the human microbiome could exceed the total number of human genes by a factor of 100-to-one. Many of these organisms have not been successfully cultured, identified, or otherwise characterized. Organisms expected to be found in the human microbiome, however, may generally be categorized as bacteria (the majority), members of domain Archaea, yeasts, and single-celled eukaryotes as well as various helminth parasites and viruses, the latter including viruses that infect the cellular microbiome organisms (e.g., bacteriophages, the viruses of bacteria). The HMP will address some of the most inspiring, vexing and fundamental scientific questions today. Importantly, it also has the potential to break down the artificial barriers between medical microbiology and environmental microbiology. It is hoped that the HMP will not only identify new ways to determine health and predisposition to diseases but also define the parameters needed to design, implement and monitor strategies for intentionally manipulating the human microbiota, to optimize its performance in the context of an individual's physiology. The HMP has been described as "a logical conceptual and experimental extension of the Human Genome Project." In 2007 the Human Microbiome Project was listed on the NIH Roadmap for Medical Research as one of the New Pathways to Discovery. Organized characterization of the human microbiome is also being done internationally under the auspices of the International Human Microbiome Consortium. The Canadian Institutes of Health Research, through the CIHR Institute of Infection and Immunity, is leading the Canadian Microbiome Initiative to develop a coordinated and focused research effort to analyze and characterize the microbes that colonize the human body and its potential alteration during chronic disease states. The HMP includes the following goals: The impact to date of the Human Microbiome Project may be partially assessed by examination of research sponsored by the HMP. Over 190 peer-reviewed publications are listed on the HMP website from June 2009 through August 2012. Major categories of work funded by HMP include: Developments funded by HMP include: On 13 June 2012, a major milestone of the Human Microbiome Project (HMP) was announced by the NIH director Francis Collins. The announcement was accompanied with a series of coordinated articles published in Nature and several journals in the Public Library of Science (PLoS) on the same day. By mapping the normal microbial make-up of healthy humans using genome sequencing techniques, the researchers of the HMP have created a reference database and the boundaries of normal microbial variation in humans. From 242 healthy U.S. volunteers, more than 5,000 samples were collected from tissues from 15 (men) to 18 (women) body sites such as mouth, nose, skin, lower intestine (stool) and vagina. All the DNA, human and microbial, were analyzed with DNA sequencing machines. The microbial genome data were extracted by identifying the bacterial specific ribosomal RNA, 16S rRNA. The researchers calculated that more than 10,000 microbial species occupy the human ecosystem and they have identified 81 – 99% of the genera. In addition to establishing the human microbiome reference database, the HMP project also discovered several "surprises", which include: Among the first clinical applications utilizing the HMP data, as reported in several PLoS papers, the researchers found a shift to less species diversity in vaginal microbiome of pregnant women in preparation for birth, and high viral DNA load in the nasal microbiome of children with unexplained fevers. Other studies using the HMP data and techniques include role of microbiome in various diseases in the digestive tract, skin, reproductive organs and childhood disorders. 
