 Oncogenomics is a relatively new sub-field of genomics that applies high throughput technologies to characterize genes associated with cancer. Oncogenomics is synonymous with "cancer genomics". Cancer is a genetic disease caused by accumulation of mutations to DNA leading to unrestrained cell proliferation and neoplasm formation. The goal of oncogenomics is to identify new oncogenes or tumor suppressor genes that may provide new insights into cancer diagnosis, predicting clinical outcome of cancers, and new targets for cancer therapies. The success of targeted cancer therapies such as Gleevec, Herceptin, and Avastin raised the hope for oncogenomics to elucidate new targets for cancer treatment. Besides understanding the underlying genetic mechanisms that initiates or drives cancer progression, one of the main goals of oncogenomics is to allow for the development of personalized cancer treatment. Cancer develops due to an accumulation of mutations in DNA. These mutations accumulate randomly, and thus, different DNA mutations and mutation combinations exist between different individuals with the same type of cancer. Thus, identifying and targeting specific mutations which have occurred in an individual patient may lead to increased efficacy of cancer therapy. The completion of the Human Genome Project has greatly facilitated the field of oncogenomics and has increased the abilities of researchers to find cancer causing genes. In addition, the sequencing technologies now available for sequence generation and data analysis have been applied to the study of oncogenomics. With the amount of research conducted on cancer genomes and the accumulation of databases documenting the mutational changes, it has been predicted that the most important cancer-causing mutations, rearrangements, and altered expression levels will be cataloged and well characterized within the next decade. Cancer research may look either on the genomic level at DNA mutations, the epigenetic level at methylation or histone modification changes, the transcription level at altered levels of gene expression, or the protein level at altered levels of protein abundance and function in cancer cells. Oncogenomics focuses on the genomic, epigenomic, and transcript level alterations in cancer. The genomics era became established with success in the 1990s, due to the generation of DNA sequences of many organisms. In the 21st century, the completion of the Human Genome Project at the Wellcome Trust Sanger Institute paved the way for many new endeavors for studying the functional genomics and examining the genomes which characterize different diseases. Cancer has been one of the main focuses. Reasons why access to whole genome sequencing is so important to cancer (or cancer genome) research: The first cancer genome was sequenced in 2008 by Timothy J. Ley et. al. This study sequenced a typical Acute Myeloid Leukaemia (AML) genome and it normal counterpart genome obtained from the same patient's skin. When comparing the two sequences these researchers discovered 10 genes which contained acquired mutations: All of these genes are involved in pathways known to contribute to cancer pathogenesis, but before this study most of these genes would not have been candidates for targeted gene therapy based on the prior understanding of cancer. Thus the results of this study was successful in showing the importantce of whole genome sequencing techniques in identifying somatic mutations involved in cancer. This study also showed the importance of parallel sequencing of the patient's normal genome to determine which mutations/variants were inherited or acquired. This technique is important in the identification of the true somatic mutations. Research examining the genomes and transcriptomes of cancer cells are currently extensively complemented by state of the art technologies. As well, the identification of the relative abundance of alternative transcripts has become an important component of the study of cancer. It has been shown that particular alternative transcript forms are correlated with specific types of cancer. With this impact, generation of exon-array technologies which are able to quantify alternate splice forms, and other transcript sequencing technologies, have become an important part of oncogenomics. With the amounts of sequencing data and expression profiling data being generated, the development of bioinformatics technologies to statistically analyze this data is essential. As well, after the identification of these oncogenes, much research still remains to be done to analyze the functional characteristics of these genes and how they contribute to the cancer phenotype. For example, examination of transformational capabilities of discovered oncogenes are important for confirming their impact in tumour formation. In addition, in cancerous cells, many DNA mutations accumulate. It is important to identify genes which are important in the early stages of cancer progression and in cancer development. Identification of mutations in these genes will be most helpful in diagnosis and in finding new targets for cancer therapy. Operomics is an approach that aims to integrate genomics, transcriptomics, and proteomics in order to achieve a complete understanding of the molecular mechanisms which underlie the development of cancer. This involves simultaneous molecular analysis of DNA, RNA, and protein of tumor tissue samples. With increasing advances in technologies to analyze cancer cells, operomics will be an overall goal of cancer research. Comparative Oncogenomics is a branch of oncogenomics which uses cross-species comparisons to identify oncogenes. This research involves studying cancer genomes, transcriptomes, and proteomes in other model organisms, such as mice, identifying potential oncogenes, and referring back to human cancer samples to see whether homologues of these oncogenes are also important in causing cancer in humans. Recent research has found that the genetic alterations in mouse models have been found to be exceptionally similar to those found in human cancers. This branch of oncogenomics useful in that different types of cancer may be studied in animal models. These models are generated by various methods, including retroviral insertion mutagenesis or graft transplantation of cancerous cells. Comparative oncogenomics is a powerful approach to oncogene identification. One approach to studying oncogenomics, which shows great promise in producing useful cancer therapies by taking advantage of mutational aberrations in cancer cells, is the strategic exploitation of synthetic lethality interactions between multiple genes. Frequently, known oncogenes may be essential for survival of all cells (not only cancer cells). Thus, drugs intended to knock out these oncogenes (and thereby kill cancer cells) may also cause serious negative effects to normal cells: i.e., significant illness may be directly induced by the cancer therapy. To generate therapies that more specifically target cancer cells, scientists are now working to systematically examine the effect of suppressing every gene in the human genome, one at a time, in combination with the presence of the cancer-associated mutation of some other gene which has previously been identified as an oncogene. This type of search can thus identify targets for cancer therapy by exploiting the mutations that are present exclusively in cancer cells; if the knockout of an otherwise nonessential gene has little or no effect on healthy cells, but is lethal to cancerous cells containing the mutated form of a given oncogene, then the system-wide suppression of the normally nonessential gene can destroy cancerous cells while leaving healthy ones intact or relatively undamaged. (The term "synthetic lethality," here, describes this sort of synergistic effect.) Success has been observed with this method both in discovering cancer targets and in developing therapies. One example is the case of PARP-1 inhibitors specifically applied to treat BRCA1/BRCA2-associated cancers. In this case, the combined presence of PARP-1 inhibition and of the cancer-associated mutations in BRCA genes is lethal only to the cancerous cells. Phase I clinical trials of this technique suggest that it may show promise in patients with BRCA1 or BRCA2 mutations, and Phase II trials are currently underway Many databases are available to cancer researchers as resources which have banked oncogenomic research data. The Cancer Genome Project is an initiative to map out all the somatic intragenic mutations in cancer. To do this, they are systematically sequencing the exons and flanking splice junctions of all the genes in the genomes of primary tumors and cancerous cell lines. COSMIC is a resource which displays the data generated from these experiments. As of February 2008, the CGP has identified 4746 genes and 2985 mutations in 1848 tumours analyzed. The Cancer Genome Anatomy Project from National Cancer Institute also has banked much information of research on cancer genome, transcriptome, and proteome. Progenetix is another oncogenomic reference database, presenting cytogenetic and molecular-cytogenetic tumor data. Oncomine has compiled data from cancer transcriptome profiles. The Integrative Oncogenomics database IntOGen integrates multidimensional human oncogenomic data classified by tissue type using the ICD-O terms. Data mining for different alteration types, such as Gene Expression and CNV are made available in the IntOGen database. The International Cancer Genome Consortium is so far the biggest project to collect human cancer genome data. The data is accessible through the ICGC website. The BioExpressÂ® Oncology Suite1 contains gene expression data from primary, metastatic, and benign tumor samples, and normal samples, including matched adjacent controls. The suite includes valuable hematological malignancy samples for many well-known cancers. Specific databases for model animals also exist, as for example the Retrovirus Tagged Cancer Gene Database (RTCGD) has compiled research on retroviral and transposon insertional mutagenesis in mouse tumors. Mutational analysis of entire gene families has been a powerful approach to oncogenomics which has been informative. Genes of the same family have similar functions, as predicted by similar coding sequences and protein domains, have been systematically sequenced in cancerous genomes to identify particular pathways which may be associated with cancer progression. One such class of families which has been studied is the kinase family genes, involved in adding phosphate groups to proteins, and phosphatase family genes, involved with removing phosphate groups from proteins. These families were first examined because of their apparent role in transducing cellular signals of cell growth or death. In particular, more than 50% of colorectal cancers were found to carry a mutation in a kinase or phosphatase gene. Phosphatidylinositold 3-kinases (PIK3CA) gene encode for lipid kinases which were identified to commonly contain mutations in colorectal, breast, gastric, lung, and various other types of cancer. Drug therapies have already been developed to inhibit PIK3CA. Another example is the BRAF gene was identified in 2004, which was one of the first genes ever to be implicated in melanomas. BRAF encodes a serine/threonine kinase which is involved in the RAS-RAF-MAPK growth signaling pathway, and they found that mutations in BRAF causing constitutive phosphorylation and activity were found in 59% of melanomas. Before BRAF, there was very little understanding of the genetic mechanism of the development of melanomas, and therefore, prognosis for patients was poor. Thus, the CGP set out to discover genes involved with melanomas and identified BRAF, which is now a target of new cancer therapies, with clinical data of BRAF inhibiting targets already generated Recent research has found links between mitochondrial DNA (mtDNA) mutations and the formation of tumors. There have been 4 different types of mtDNA mutations that have been identified: Point mutations have been observed in the coding and non-coding region of the mtDNA contained in cancerous cells. In individuals with bladder, head and neck, and lung cancers the point mutations within the coding region showed signs of being homoplasmic (resembling each other). This suggested hat when a healthy cell transforms into a tumor cell (i.e. goes through a neoplastic transformation) then the mitochondria seem to become homogenous (the same). There was also a high prevalence of point mutations located within the non-coding region, D-loop, of the cancerous mitochondria suggesting that mutations within this region might also be an important characteristic in some cancers. This type of mutation is sporadically detected due to their small size ( < 1kb). The appearance of certain specific mtDNA mutations (264-bp deletion and 66-bp deletion in the complex 1 subunit gene ND1) in multiple types of cancer provide some evidence that small mtDNA deletions might appear at the beginning of tumorigenesis. It also suggests that the amount of mitochondria containing these deletions will increase as the tumor progresses. There is an exception to this though. There is a relatively large deletion that has been found to appear in a multitude of different cancers (known as the "common deletion"), but it has been observed that more mtDNA large scale deletions have been found in normal cells compared to tumor cells. It is believed that this is due to a seemingly adaptive process of tumor cells to eliminate any mitochondria which contain these large scale deletions (the "common deletion" is > 4kb). Two small mtDNA insertions of ~260 and ~520 bp have been found to be present in Breast cancer, Gastric cancer, Hepatocellular carcinoma (HCC), and Colon cancer but such changes have also been found in normal healthy cells. Therefore a correlation between these insertions and cancer has yet to be established. The characterization of mtDNA by using real-time PCR assays showed a large presence of quantitative alteration of mtDNA copy number in a wide range of cancers when compared to normal cells. This increase in copy number is expected to occur because of oxidative stress. On the other hand, the decrease is thought to be caused by somatic point mutations in the replication origin site of the H-strand and/or the D310 homopolymeric c-stretch in the D-loop region, mutations in the p53 (tumor suppressor gene) mediated pathway, and/or inefficient enzyme activity due to POLG mutations. Although there is an increase/decrease in copy number it does remain constant within tumor cells. The fact that the amount of mtDNA is maintained as constant in tumor cells when compared to non-tumor cells suggests that the amount of mtDNA is controlled by a much more complicated system in tumor cells, rather than simply altered as a consequence of abnormal cell proliferation. The role of mtDNA content in human cancers is estimated to have some degree of specificity for particular tumor types or sites. Above Table: Mutations in mitochondrial DNA in various cancers. This table only includes a few of the cancers that were looked at. One of the studies included which looked at the location of point mutations in different cancers, contained 867 patients and 23 different types of cancers. 57.7% (500/867) contained somatic point putations, and of the 1172 mutations surveyed 37.8% (443/1127) were located in the D-loop control region, 13.1% (154/1172) were located in the tRNA or rRNA genes, and 49.1% (575/1127) were found in the mRNA genes needed for producing complexes required for mitochondrial respiration. Currently anticancer drugs have been manufactured to target mtDNA and have shown positive results in killing tumor cells. There has also been research done in using mitochondrial mutations as biomarkers for cancer cell therapy. It is easier to target mutation within the mitochondrial DNA as opposed to nuclear DNA because the mitochondrial genome is much smaller and therefore easier to screen for specific mutations. It is also thought that the mtDNA content alterations found in blood samples might be able to serve as a screening marker for predicting future cancer susceptibility as well as tracking malignant tumor progression. Along with these potential helpful characteristics of mtDNA, it is also not under the control of the cell cycle and it is important for maintaining ATP generation and mitochondrial homeostasis. These characteristics make targeting mtDNA a practical therapeutic strategy. With advances in this field, scientists are discovering potential biomarkers for use in cancer staging, prognosis, and treatment. There are several different biomarkers that can be used for these purposes. They can range from single-nucleotide polymorphisms (SNPs), chromosomal aberrations, changes in DNA copy number, mirosatellite instability, differential promoter region methylation, or even high or low protein levels. The identification of characteristics associated with various cancers has the possibility for more effective personalized treatment options by developing drugs designed to target the biomarkers which are present in an individual patient. For a table that outlines some cancers and their respective biomarkers see the Cancer Biomakers page. 
