 Transmission electron microscopy DNA sequencing is an emerging third-generation, single-molecule sequencing technology that uses transmission electron microscopy techniques. DNA is visible under the electron microscope; however, it must be labeled with heavy atoms so that the DNA bases can be clearly visualized. In addition, specialized imaging techniques and aberration corrected optics are beneficial for obtaining the resolution required to image the labeled DNA molecule. Transmission electron microscopy DNA sequencing advantageously may provide extremely long read lengths, but it is not yet commercially available. Only a few years after James Watson and Francis Crick deduced the structure of DNA, and nearly two decades before Frederick Sanger published the first method for rapid DNA sequencing, Richard Feynman, an American physicist, envisioned the electron microscope as the tool that would one day allow biologists to “see the order of bases in the DNA chain”. Feynman believed that if the electron microscope could be made powerful enough, then it would become possible to visualize the atomic structure of any and all chemical compounds, including DNA. To this day, despite the invention of a multitude of chemical and fluorescent sequencing technologies, microscopy is still being explored as a means of performing single-molecule DNA sequencing. Two biotechnology companies have conceived of methods for high throughput, direct detection of DNA bases by transmission electron microscopy; however, these studies are still in their infancy and are far from being commercially available. The following progress in these technologies has been reported: The electron microscope has the capacity to obtain a resolution of up to 100 pm, whereby microscopic biomolecules and structures such as viruses, ribosomes, proteins, lipids, small molecules and even single atoms can be observed. Although DNA is visible when observed with the electron microscope, the resolution of the image obtained is not high enough to allow for deciphering the sequence of the individual bases, i.e., DNA sequencing. However, upon differential labeling of the DNA bases with heavy atoms or metals, it is possible to both visualize and distinguish between the individual bases. Therefore, electron microscopy in conjunction with differential heavy atom DNA labeling could be used to directly image the DNA in order to determine its sequence. As in a standard polymerase chain reaction (PCR), the double stranded DNA molecules to be sequenced must be denatured before the second strand can be synthesized with labeled nucleotides. The elements that make up biological molecules (C, H, N, O, P, S) are too light (low atomic number, Z) to be clearly visualized as individual atoms by transmission electron microscopy. To circumvent this problem, the DNA bases can be labeled with heavier atoms (higher Z). Each nucleotide is tagged with a characteristic heavy label, so that they can be distinguished in the transmission electron micrograph. The DNA molecules must be stretched out on a thin, solid substrate so that order of the labeled bases will be clearly visible on the electron micrograph. Molecular combing is a technique that utilizes the force of a receding air-water interface to extend DNA molecules, leaving them irreversibly bound to a silane layer once dry. This is one means by which alignment of the DNA on a solid substrate may be achieved. Transmission electron microscopy (TEM) produces high magnification, high resolution images by passing a beam of electrons through a very thin sample. Whereas atomic resolution has been demonstrated with conventional TEM, further improvement in spatial resolution requires correcting the spherical and chromatic aberrations of the microscope lenses. This has only been possible in scanning transmission electron microscopy where the image is obtained by scanning the object with a finely focused electron beam, in a way similar to a cathode ray tube. However, the achieved improvement in resolution comes together with irradiation of the studied object by much higher beam intensities, the concomitant sample damage and the associated imaging artefacts. Different imaging techniques are applied depending on whether the sample contains heavy or light atoms: Dark and bright spots on the electron micrograph, corresponding to the differentially labeled DNA bases, are analyzed by computer software. Transmission electron microscopy DNA sequencing is not yet commercially available, but the long read lengths that this technology may one day provide will make it useful in a variety of contexts. When sequencing a genome, it must be broken down into pieces that are short enough to be sequenced in a single read. These reads must then be put back together like a jigsaw puzzle by aligning the regions that overlap between reads; this process is called de novo genome assembly. The longer the read length that a sequencing platform provides, the longer the overlapping regions, and the easier it is to assemble the genome. From a computational perspective, microfluidic Sanger sequencing is still the most effective way to sequence and assemble genomes for which no reference genome sequence exists. The relatively long read lengths provide substantial overlap between individual sequencing reads, which allows for greater statistical confidence in the assembly. In addition, long Sanger reads are able to span most regions of repetitive DNA sequence which otherwise confound sequence assembly by causing false alignments. However, de novo genome assembly by Sanger sequencing is extremely expensive and time consuming. Second generation sequencing technologies, while less expensive, are generally unfit for de novo genome assembly due to short read lengths. In general, third generation sequencing technologies, including transmission electron microscopy DNA sequencing, aim to improve read length while maintaining low sequencing cost. Thus, as third generation sequencing technologies improve, rapid and inexpensive de novo genome assembly will become a reality. A haplotype is a series of linked alleles that are inherited together on a single chromosome. DNA sequencing can be used to genotype all of the single nucleotide polymorphisms (SNPs) that constitute a haplotype. However, short DNA sequencing reads often cannot be phased; that is, heterozygous variants cannot be confidently assigned to the correct haplotype. In fact, haplotyping with short read DNA sequencing data requires very high coverage (average >50x coverage of each DNA base) to accurately identify SNPs, as well as additional sequence data from the parents so that Mendelian transmission can be used to estimate the haplotypes. Sequencing technologies that generate long reads, including transmission electron microscopy DNA sequencing, can capture entire haploblocks in a single read. That is, haplotypes are not broken up among multiple reads, and the genetically linked alleles remain together in the sequencing data. Therefore, long reads make haplotyping easier and more accurate, which is beneficial to the field of population genetics. Genes are normally present in two copies in the diploid human genome; genes that deviate from this standard copy number are referred to as copy number variants (CNVs). Copy number variation can be benign (these are usually common variants, called copy number polymorphisms) or pathogenic. CNVs are detected by fluorescence in situ hybridization (FISH) or comparative genomic hybridization (CGH). To detect the specific breakpoints at which a deletion occurs, or to detect genomic lesions introduced by a duplication or amplification event, CGH can be performed using a tiling array (array CGH), or the variant region can be sequenced. Long sequencing reads are especially useful for analyzing duplications or amplifications, as it is possible to analyze the orientation of the amplified segments if they are captured in a single sequencing read. Cancer genomics, or oncogenomics, is an emerging field in which high-throughput, second generation DNA sequencing technology is being applied to sequence entire cancer genomes. Analyzing this short read sequencing data encompasses all of the problems associated with de novo genome assembly using short read data. Furthermore, cancer genomes are often aneuploid. These aberrations, which are essentially large scale copy number variants, can be analyzed by second-generation sequencing technologies using read frequency to estimate the copy number. Longer reads would, however, provide a more accurate picture of copy number, orientation of amplified regions, and SNPs present in cancer genomes. The microbiome refers the total collection of microbes present in a microenvironment and their respective genomes. For example, an estimated 100 trillion microbial cells colonize the human body at any given time. The human microbiome is of particular interest, as these commensal bacteria are important for human health and immunity. Most of the Earth's bacterial genomes have not yet been sequenced; undertaking a microbiome sequencing project would require extensive de novo genome assembly, a prospect which is daunting with short read DNA sequencing technologies. Longer reads would greatly facilitate the assembly of new microbial genomes. Compared to other second- and third-generation DNA sequencing technologies, transmission electron microscopy DNA sequencing has a number of potential key strengths and weaknesses, which will ultimately determine its usefulness and prominence as a future DNA sequencing technology. Many non-Sanger second- and third-generation DNA sequencing technologies have been or are currently being developed with the common aim of increasing throughput and decreasing cost such that personalized genetic medicine can be fully realized. Both the US$10 million Archon X Prize for Genomics supported by the X Prize Foundation (Santa Monica, CA, USA) and the US$70 million in grant awards supported by the National Human Genome Research Institute of the National Institutes of Health (NIH-NHGRI) are fueling the rapid burst of research activity in the development of new DNA sequencing technologies. Since different approaches, techniques, and strategies are what define each DNA sequencing technology, each has its own strengths and weaknesses. Comparison of important parameters between various second- and third-generation DNA sequencing technologies are presented in Table 1. 
